import-scripts/clone_db_wrapper.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Script for running pre-import steps
+# Consists of the following:
+# - Determine which database is "production" vs "not production"
+# - Drop tables in the non-production database
+# - Clone the production database into the non-production database
+
+PORTAL_SCRIPTS_DIRECTORY=$1
+if [ -z $PORTAL_SCRIPTS_DIRECTORY ]; then
+    PORTAL_SCRIPTS_DIRECTORY="/data/portal-cron/scripts"
+fi
+if [ ! -f $PORTAL_SCRIPTS_DIRECTORY/automation-environment.sh ] ; then
+    echo "`date`: Unable to locate automation_env, exiting..."
+    exit 1
+fi
+source $PORTAL_SCRIPTS_DIRECTORY/automation-environment.sh
+
+# Create tmp directory for processing
+tmp=$PORTAL_HOME/tmp/import-cron-genie
+if ! [ -d "$tmp" ] ; then
+    if ! mkdir -p "$tmp" ; then
+        echo "Error : could not create tmp directory '$tmp'" >&2
+        exit 1
+    fi
+fi
+if [[ -d "$tmp" && "$tmp" != "/" ]]; then
+    rm -rf "$tmp"/*
+fi
+
+# This file will be used throughout entire import job to determine directionality
```

Comment:
I think this comment is out of date or misplaced?

Reply:
good catch, thanks!